src/MasterPeace/Bundle/UserBundle/Controller/ClassroomController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace MasterPeace\Bundle\UserBundle\Controller;
+
+use MasterPeace\Bundle\UserBundle\Entity\Classroom;
+use MasterPeace\Bundle\UserBundle\Form\ClassroomType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ClassroomController extends Controller
+{
+    /**
+     * @Route ("/list", name="classroom_list")
+     *
+     * @return Response
+     */
+    public function listAction()
+    {
+        $em = $this
```

Comment:
$this ->getDoctrine()->getManager(); sita veiksma jau du kart naudoji klaseje. As siulyciau ClassroomRepository kaip servisa pasidaryti ir kontroleryje is kontainerio pasiimti